Source/ASTextNode2.mm

```diff
@@ -30,18 +29,11 @@
 
 #import <AsyncDisplayKit/CoreGraphics+ASConvenience.h>
 #import <AsyncDisplayKit/ASObjectDescriptionHelpers.h>
+#import <AsyncDisplayKit/ASTextCacheKey.h>
 #import <AsyncDisplayKit/ASTextLayout.h>
+#import <AsyncDisplayKit/ASCache.h>
 #import <AsyncDisplayKit/ASThread.h>
 
-@interface ASTextCacheValue : NSObject {
-  @package
-  ASDN::Mutex _m;
-  std::deque<std::tuple<CGSize, ASTextLayout *>> _layouts;
-}
-@end
-@implementation ASTextCacheValue
-@end
-
 /**
  * If set, we will record all values set to attributedText into an array
```

Comment:
It would be great to use this feature to collect a bunch of data and add it to our unit tests! This might collect some noise when iOS changes, but it would also be an excellent way to identify breakage / race conditions from core text classes changing (e.g. if we had an auto-scrolling collection showing ASTextCellNodes, measuring them in batches of 20 insertions to the data source).

Reply:
Yes we've actually got some – `AttributedStringsFixture0.plist` that I captured from Pinterest a while back. Just need to build a test that uses them.